City-of-Helsinki/berth-reservations-admin
# VEN-553 | Style fixes

## Description :sparkles:
- Add a wrapper to the harbor's image
- Set whitespace to `nowrap` for `Chip` component

## Issues :bug:
- Harbor's image in the single harbor page is stretched in Safari
![image](https://user-images.githubusercontent.com/23040926/78036588-25c01500-7373-11ea-855b-dd21b86328cb.png)
- On small screens, the status label is wrapped (applications page)
![image](https://user-images.githubusercontent.com/23040926/78036731-5142ff80-7373-11ea-84fd-080c1393840f.png)

### Closes :no_good_woman:
[VEN-553](https://helsinkisolutionoffice.atlassian.net/browse/VEN-553): Harbor header's image on Safari & Status label

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

### Manual testing :construction_worker_man:
Testing the harbor's image:
- On Safari browser
- Go the harbors page, click on one harbor
- The image dimensions should follow the design shown in the Screenshots below

Testing the status text:
- Go to the applications page
- Resize your window to be relatively small 
- The status label should remain on the same line

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/78037777-a5021880-7374-11ea-8f2d-1325a21ab3f5.png)

![image](https://user-images.githubusercontent.com/23040926/78037523-5b193280-7374-11ea-82e0-fe4ce31b1d38.png)


## Additional notes :spiral_notepad:
